src/rf/js/src/home/components/modals.js

```diff
@@ -293,8 +327,12 @@ var UploadModal = React.createBackboneClass({
             }
         }
 
-        if (!layerData.area) {
-            errors.area = ['Please enter an area.'];
+        if (layerData.capture_start && !layerData.capture_end) {
+            errors.capture_end.push('Please enter a date.');
+        }
+
+        if (layerData.capture_end && !layerData.capture_start) {
+            errors.capture_start.push('Please enter a date.');
         }
```

Comment:
Since we don't use dates for anything, the purpose of this validation is unclear. Actually, I'm not sure why they were required fields to begin with.